---
panda_vision/BlockPose[] poses